Automattic/jetpack
# Move Whatsapp styles to Send-a-Message directory

https://github.com/Automattic/jetpack/pull/18487 moved the required plan check from the Send-a-Message block down to the WhatsApp block. In order to do this, it was necessary to change the block's entry in `jetpack/extensions/index.json` from `send-a-message/whatsapp-button` to `whatsapp-button`, so that it would match the slug used in registering the block on the backend here: https://github.com/Automattic/jetpack/blob/7a208dc8117f025d9846501ae49ed97a9a9aaa05/projects/plugins/jetpack/extensions/blocks/send-a-message/whatsapp-button/whatsapp-button.php#L24

This breaks the CSS, because the [code that loads in styles](https://github.com/Automattic/jetpack/blob/7a208dc8117f025d9846501ae49ed97a9a9aaa05/projects/plugins/jetpack/class.jetpack-gutenberg.php#L569) expects the path to the `view.scss` file to match the block name, so `send-a-message/whatsapp-button` in this case.

Essentially there are competing naming conventions here. This PR proposes a quick fix by moving the styles up to the parent block `Send-a-Message` directory, which does not have the same naming issue, and loads it from there.




Fixes https://github.com/Automattic/jetpack/issues/18940

#### Changes proposed in this Pull Request:

*

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






On Jetpack, you will need to pull the branch and run `yarn run build` before testing.
On wpcom, test by applying the diff.

* Create a new post and add a WhatsApp button
* View the post on the frontend and verify that styles have loaded in
* Follow instructions from https://github.com/Automattic/jetpack/pull/18487 to verify that the upgrade nudges are still working.


#### Proposed changelog entry for your changes:


*
